ci-operator/step-registry/ipi/conf/vsphere/vips/ipi-conf-vsphere-vips-commands.sh

```diff
@@ -14,6 +14,7 @@ third_octet=$(grep -oP 'ci-segment-\K[[:digit:]]+' <(echo "${LEASED_RESOURCE}"))
 
 echo "192.168.${third_octet}.2" >> "${SHARED_DIR}"/vips.txt
 echo "192.168.${third_octet}.3" >> "${SHARED_DIR}"/vips.txt
+echo "192.168.${third_octet}.0/27" >> "${SHARED_DIR}"/machinenetworks.txt
```

Comment:
Use `machinecidr.txt` to match more closely with what the text in the file contains.
```suggestion
echo "192.168.${third_octet}.0/27" >> "${SHARED_DIR}"/machinecidr.txt
```